flyway/src/main/scala/com/avast/sst/flyway/FlywayConfig.scala

```diff
@@ -0,0 +1,24 @@
+package com.avast.sst.flyway
+
+final case class FlywayConfig(url: String,
+                              username: String,
+                              password: String,
+                              baselineOnMigrate: Boolean = false,
+                              baselineVersion: Option[String] = None,
```

Comment:
could we use `org.flywaydb.core.api.MigrationVersion` instead of String?

Reply:
I agree with `Charset` but `MigrationVersion` is really just a wrapper around `String`, it nevers fails , it wraps any string, so I don't see the benefit of having to create a `ConfigReader` for this because no validation will performed, what do you think?

Reply:
:thinking: 
hmm, you're right that it's not super strongly typed, but still, you can use [`MigrationVersion.LATEST`](https://github.com/flyway/flyway/blob/6e2e4ed16651bcb0c15366863a1912a8c0574942/flyway-core/src/main/java/org/flywaydb/core/api/MigrationVersion.java#L37)